### ISSUE TITLE
Could not find file libncursesw6

### ISSUE THREAD

**Issue by burnsba** [MV10/dotnet-curses#18]
Trying to run on windows 11, dotnet 8, the library throws an exception on initialization.

dotnet --list-runtimes
...
Microsoft.WindowsDesktop.App 8.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

dotnet --list-sdks
...
8.0.100 [C:\Program Files\dotnet\sdk]

--

Exception:
```
Unhandled exception. System.TypeInitializationException: The type initializer for 'Mindmagma.Curses.Interop.Native' threw an exception.
 ---> System.TypeInitializationException: The type initializer for 'Mindmagma.Curses.Interop.NCursesLibraryHandle' threw an exception.
 ---> System.IO.FileNotFoundException: Could not find or load the native library from any name: [ libncursesw6 ]
   at NativeLibraryLoader.LibraryLoader.LoadNativeLibrary(String[] names, PathResolver pathResolver)
   at NativeLibraryLoader.NativeLibrary..ctor(String[] names, LibraryLoader loader, PathResolver pathResolver)
   at NativeLibraryLoader.NativeLibrary..ctor(String[] names)
   at Mindmagma.Curses.Interop.NCursesLibraryHandle.FindLibrary()
   at Mindmagma.Curses.Interop.NCursesLibraryHandle..cctor()
   --- End of inner exception stack trace ---
   at Mindmagma.Curses.Interop.Native.NativeToDelegate[D](String exportedFunctionName)
   at Mindmagma.Curses.Interop.Native..cctor()
   --- End of inner exception stack trace ---
   at Mindmagma.Curses.Interop.Native.initscr()
   at Mindmagma.Curses.NCurses.InitScreen()
   at ConsoleApp1.Program.Main(String[] args) in C:\Users\bburns\code\scratch\abq\ConsoleApp1\Program.cs:line 9
```

Program.cs

```csharp
using Mindmagma.Curses;

namespace ConsoleApp1
{
    internal class Program
    {
        static void Main(string[] args)
        {
            var Screen = NCurses.InitScreen();
        }
    }
}

```

csproj file
```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net8.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
    <PlatformTarget>x64</PlatformTarget>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="dotnet-curses" Version="1.0.3" />
  </ItemGroup>

</Project>

```

**Comment by MV10** [comment]
Hi, I have to start with the obvious question -- did you download the curses library?

From the [README](https://github.com/MV10/dotnet-curses?tab=readme-ov-file#the-native-library):

> For Windows, download it from Thomas Dickey’s site (the current ncurses maintainer) from the 64-bit link under the "MinGW Ports" heading on [this page](https://invisible-island.net/ncurses/#download_mingw), and extract the DLLs anywhere in your `%PATH%`.

**Comment by burnsba** [comment]
Ok thank you. I read the readme, but at no point was it made clear downloading the dll was required for windows.

> For Windows, download it

Download "it"? What are we downloading, and why does it matter? You might want to update the readme to make that more clear. I initially thought this nuget package was self contained, if I had realized it wasn't, I wouldn't have used it in the first place.

Anyways, once I copied the dll to the bin folder I was able to run the above program without it throwing an exception.
